backend/src/src-licenses/src/main/java/org/eclipse/sw360/licenses/LicenseHandler.java

```diff
@@ -265,6 +313,14 @@ public RequestSummary importAllSpdxLicenses(User user) throws TException {
         return handler.importAllSpdxLicenses(user);
     }
 
+    @Override
+    public RequestSummary importAllOSADLLicenses(User user) throws TException {
+        if(! PermissionUtils.isUserAtLeast(UserGroup.CLEARING_ADMIN, user)){
```

Comment:
Indentation after if and before ( same before the {, please look into the code for similar indentation issue,most of the indentation issue i have highlighted.